packages/mui-joy/src/FormControl/FormControl.tsx

```diff
@@ -11,10 +11,11 @@ import formControlClasses, { getFormControlUtilityClass } from './formControlCla
 import { FormControlProps, FormControlOwnerState, FormControlTypeMap } from './FormControlProps';
 
 const useUtilityClasses = (ownerState: FormControlOwnerState) => {
-  const { disabled, error, size, color } = ownerState;
+  const { disabled, error, size, color, orientation } = ownerState;
```

Comment:
Does `FormControl` need `orientation` prop?

Reply:
FYI, I didn't add the prop. The prop was already there. I only added a classname for it for consistency.

<img width="676" alt="Screenshot 2023-01-05 at 13 03 39" src="https://user-images.githubusercontent.com/32841130/210849406-72fee725-a143-4aa1-b93e-ef5273c78330.png">